### PR TITLE
Updated import statements in the tutorial

### PR DESCRIPTION
**Pull request by jesper-friis**
Simple PR correcting the tutorial.

Maybe we could consider to make a test that run doctest on it to ensure that it actually works.

### CODE REVIEW

**Review by CasperWA — APPROVED** [review]
Does `doctest` run the Python code? What is `doctest`?

File: docs/tutorial.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ New triples can be added either with the `parse()` method (for backends that sup
 ```python
 # en(msg) is a convenient function for adding english literals.
 # It is equivalent to ``triplestore.Literal(msg, lang="en")``.
```
Comment (CasperWA) [inline docs/tutorial.md:54]:
```suggestion
# It is equivalent to ``tripper.triplestore.Literal(msg, lang="en")``.
```

**Review by CasperWA — CHANGES_REQUESTED** [review]

**Comment by jesper-friis** [comment]
> Does `doctest` run the Python code? What is `doctest`?

[doctest](https://docs.python.org/3/library/doctest.html) runs Python code in docstrings. Good for ensuring that the documentation is up-to-date and that examples actually works.

**Comment by CasperWA** [comment]
> > Does `doctest` run the Python code? What is `doctest`?
> 
> [doctest](https://docs.python.org/3/library/doctest.html) runs Python code in docstrings. Good for ensuring that the documentation is up-to-date and that examples actually works.

It seems to only work for Python shell doc strings (starting with `>>>`). Perhaps this one will be better: https://github.com/tmarktaylor/phmdoctest ?

**Review by CasperWA — APPROVED** [review]

**Comment by CasperWA** [comment]
Squash+merge at will.

**Pull request merged**